tools/openms/CometAdapter.xml

```diff
@@ -83,54 +82,55 @@ python3 '$__tool_directory__/fill_ctd.py' '@EXECUTABLE@.ctd' '$args_json' '$hard
-    <param argument="-min_peptide_length" type="integer" optional="true" min="5" max="63" value="5" label="Minimum peptide length to conside" help=""/>
-    <param argument="-max_peptide_length" type="integer" optional="true" min="5" max="63" value="63" label="Maximum peptide length to conside" help=""/>
-    <param argument="-num_hits" type="integer" optional="true" value="1" label="Number of peptide hits in output file" help=""/>
-    <param argument="-precursor_charge" type="text" optional="true" value="0:0" label="Precursor charge range to search (if spectrum is not annotated with a charge or if override_charge!=keep any known): 0:[num] == search all charges, 2:6 == from +2 to +6, 3:3 == +3" help="">
+    <param argument="-missed_cleavages" type="integer" optional="false" min="0" max="5" value="1" label="Number of possible cleavage sites missed by the enzyme" help="It has no effect if enzyme is unspecific cleavage"/>
+    <param argument="-min_peptide_length" type="integer" optional="false" min="5" max="63" value="5" label="Minimum peptide length to conside" help=""/>
+    <param argument="-max_peptide_length" type="integer" optional="false" min="5" max="63" value="63" label="Maximum peptide length to conside" help=""/>
+    <param argument="-num_hits" type="integer" optional="false" value="1" label="Number of peptide hits (PSMs) per spectrum in output file" help=""/>
+    <param argument="-precursor_charge" type="text" optional="false" value="0:0" label="Precursor charge range to search (if spectrum is not annotated with a charge or if override_charge!=keep any known): 0:[num] == search all charges, 2:6 == from +2 to +6, 3:3 == +3" help="">
       <expand macro="list_string_san" name="precursor_charge"/>
     </param>
-    <param argument="-override_charge" type="select" optional="true" label="_keep any known_: keep any precursor charge state (from input), _ignore known_: ignore known precursor charge state and use precursor_charge parameter, _ignore outside range_: ignore precursor charges outside precursor_charge range, _keep known search unknown_: keep any known precursor charge state" help="For unknown charge states, search as singly charged if there is no signal above the precursor m/z or use the precursor_charge range">
+    <param argument="-override_charge" type="select" optional="false" display="radio" label="_keep any known_: keep any precursor charge state (from input), _ignore known_: ignore known precursor charge state and use precursor_charge parameter, _ignore outside range_: ignore precursor charges outside precursor_charge range, _keep known search unknown_: keep any known precursor charge state" help="For unknown charge states, search as singly charged if there is no signal above the precursor m/z or use the precursor_charge range">
```

Comment:
Not sure why we use radio now ... not opposed just wanted to flag if this was a mistake.

Reply:
Mistake. Should be fixed in https://github.com/galaxyproteomics/tools-galaxyp/pull/697/commits/39d6e0d508a11b6699b7c464df85d1555a65ab66